src/coreclr/vm/ClrEtwAll.man

```diff
@@ -7483,7 +7483,7 @@
                             <opcode name="GCHeapDumpObjectReference" message="$(string.MonoProfilerPublisher.GCHeapDumpObjectReferenceOpcodeMessage)" symbol="CLR_MONO_PROFILER_GC_HEAP_DUMP_OBJECT_REFERENCE_OPCODE" value="70" />
                             <opcode name="MonitorContention" message="$(string.MonoProfilerPublisher.MonitorContentionOpcodeMessage)" symbol="CLR_MONO_PROFILER_MONITOR_CONTENTION_OPCODE" value="71" />
                             <opcode name="MonitorFailed" message="$(string.MonoProfilerPublisher.MonitorFailedOpcodeMessage)" symbol="CLR_MONO_PROFILER_MONITOR_FAILED_OPCODE" value="72" />
-                            <opcode name="MonitorAquired" message="$(string.MonoProfilerPublisher.MonitorAquiredOpcodeMessage)" symbol="CLR_MONO_PROFILER_MONITOR_AQUIRED_OPCODE" value="73" />
+                            <opcode name="MonitorAcquired" message="$(string.MonoProfilerPublisher.MonitorAcquiredOpcodeMessage)" symbol="CLR_MONO_PROFILER_MONITOR_AQUIRED_OPCODE" value="73" />
```

Comment:
@brianrob Is it ok to fix typos in opcode names here or would that be a breaking change?

Reply:
@am11 Could you please revert the change in the opcode names for now?

Reply:
Maybe open an issue on it so that it is not forgotten.

Reply:
Reverted and opened https://github.com/dotnet/runtime/issues/72324.

Reply:
Changing the opcode name could be considered a breaking change, depending upon how the events are consumed.  I would not expect the string id to be a breaking change unless someone was looking for a specific string that they've hardcoded somewhere, but I wouldn't worry about that.  For the most part, we have not fixed typos for opcodes, keywords, event names, and fields to avoid these types of breaks since there are lots of consumers out there that use string values instead of integer ids.